package.json

```diff
@@ -22,8 +23,10 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "test": "react-scripts test --watchAll",
     "eject": "react-scripts eject",
-    "test": "react-scripts test --watchAll"
+    "coverage": "react-scripts test --coverage  && nyc report --reporter=text-lcov ",
```

Comment:
Hello Mr. @izabayogit, Please may use already existing tests scripts from develop branch to set Travis configurations.